developer/styleguide.md

```diff
@@ -4,25 +4,24 @@ Our rules are similar to AirBnB or Meteor rules, [standard template of ESLint ru
 
 A couple of notable Reaction specific style rules:
 
-* Double quoted strings
-* Well spaced function
-* 160 character line length
-* `import` order
-  * React npm packages (`React`, `prop-types`, etc...)
-  * other npm packages
-  * meteor core packages
-  * meteor (Atmosphere) packages
-  * local app files
+-   Double quoted strings
+-   Well spaced function
+-   160 character line length
```

Comment:
Didn't we change the line length to 120? Do we want to add "spaces around brackets"?